### ISSUE TITLE
Disable progress bar and lightning_logs

### ISSUE THREAD

**Issue by ZhikangLai** [Nixtla/neuralforecast#963]
### Description

Hello there. Neuralforecast is excellent package. But I think some functions need to be improved. For instance, When I call the AutoModel,such as AutoRNN, and I will out lots of training message(train bar). Not everyone needs this information. So I think the creator of this toolbox can add this feature which can let user choose for themselves whether to ignore information about the training process.

### Use case

h = 1
models = [AutoRNN(h = h, loss=MSE(), num_samples=10)]
auto_nf = NeuralForecast(models=models, freq='D',local_scaler_type = 'minmax')
auto_cv_df = auto_nf.cross_validation(df=No4, refit=0, n_windows = len(test_data))

and it will outbreak lots of unnecessary information for me:
![1712739164275](https://github.com/Nixtla/neuralforecast/assets/110596898/67e3bd6a-4c3f-442a-8065-f3563b1f99bd)

**Comment by ZhikangLai** [comment]
additionally, I want to know "lightning_logs" folder is useful? it took up quite a lot of space in my computer
![1712740098913](https://github.com/Nixtla/neuralforecast/assets/110596898/548dee5e-e467-4224-a7f8-93b55328ded7)

**Comment by jmoralez** [comment]
Hey. We use [pytorch lightning's Trainer](https://lightning.ai/docs/pytorch/stable/common/trainer.html) which sets those defaults. You can get rid of the progress bar by passing `enable_progress_bar=False` through the config. The lightning_logs directory also comes from that and it saves the losses during training (which you can then visualize through tensorboard) and you can get rid of those by passing `logger=False` through the config.

So by changing the code to the following you shouldn't see those anymore:
```python
rnn_config = {
    **AutoRNN.get_default_config(h=h, backend='ray'),
    'enable_progress_bar': False,
    'logger': False,
}
models = [AutoRNN(h=h, loss=MSE(), config=rnn_config, num_samples=10)]
```

**Comment by ZhikangLai** [comment]
> Hey. We use [pytorch lightning's Trainer](https://lightning.ai/docs/pytorch/stable/common/trainer.html) which sets those defaults. You can get rid of the progress bar by passing `enable_progress_bar=False` through the config. The lightning_logs directory also comes from that and it saves the losses during training (which you can then visualize through tensorboard) and you can get rid of those by passing `logger=False` through the config.
> 
> So by changing the code to the following you shouldn't see those anymore:
> 
> ```python
> rnn_config = {
>     **AutoRNN.get_default_config(h=h, backend='ray'),
>     'enable_progress_bar': False,
>     'logger': False,
> }
> models = [AutoRNN(h=h, loss=MSE(), config=rnn_config, num_samples=10)]
> ```

Thank you very much for your response. That's working.
